cargo-dist/src/lib.rs

```diff
@@ -416,6 +424,23 @@ fn generate_fake_pkg(
     Ok(())
 }
 
+fn generate_omnibor_artifact_id(src_path: &Utf8Path, dest_path: &Utf8Path) -> DistResult<()> {
+    let mut cmd = Cmd::new("omnibor", "generate an OmniBOR Artifact ID");
```

Comment:
How would you feel about adding `omnibor` to the `Tools` struct? That would make it easier for us to handle the "omnibor not installed" case: https://github.com/axodotdev/cargo-dist/blob/main/cargo-dist/src/tasks.rs#L264-L282